examples/jsm/misc/VolumeSlice.d.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  Mesh,
+} from '../../../src/Three';
+
+import { volume } from './Volume';
+import { Matrix3 } from '../../../build/three.module';
```

Comment:
This looks strange. You can put `Matrix3` into the first import statement at the top instead. So:
```ts
import {
  Matrix3,
  Mesh
} from '../../../src/Three';
```

Reply:
Oops, I missed that. It was the VScode auto import function. Mostly it gets things right, sometimes... Not so much. 

I'll fix that in the morning.